dev/io.openliberty.persistence.3.2.thirdparty/resources/org/eclipse/persistence/exceptions/i18n/OptimisticLockExceptionResource.nlsprops

```diff
@@ -47,3 +47,5 @@
 
 5011=One or more objects cannot be updated because it has changed or been deleted since it was last read
 
+5012=One or more objects of class {0} with parameters list {1} cannot be updated for SQL query {2} because it has changed or been deleted since it was last read
```

Comment:
Change to:
One or more objects of the {0} class with the {1} parameter list cannot be updated for the {2} SQL query because it  changed or was deleted since it was last read